pkg/apis/networking/v1alpha1/ingress_lifecycle.go

```diff
@@ -73,11 +77,16 @@ func (is *IngressStatus) MarkLoadBalancerReady(lbs []LoadBalancerIngressStatus,
 
 // MarkLoadBalancerPending marks the "IngressConditionLoadBalancerReady" condition to unknown to
```

Comment:
```suggestion
// MarkLoadBalancerNotReady marks the "IngressConditionLoadBalancerReady" condition to unknown to
```

Reply:
Do we need to make status as **NotReady** or **Unknown**?

Reply:
Thank you for the review.

> Do we need to make status as NotReady or Unknown?

You mean that it is strange as `MarkLoadBalancerNotReady()` marks `Unknown`, correct? I thought so first, but as per https://github.com/knative/serving/issues/5076#issuecomment-519295908,

```
MarkFooNotReady -> Ready=Unknown
```

is the most prevalent for now.

Reply:
> You mean that it is strange as `MarkLoadBalancerNotReady()` marks `Unknown`, correct? I thought so first, but as per [#5076 (comment)](https://github.com/knative/serving/issues/5076#issuecomment-519295908),
> 
> ```
> MarkFooNotReady -> Ready=Unknown
> ```
> 
> is the most prevalent for now.

Yes felt strange.

Thank you i dint see https://github.com/knative/serving/issues/5076#issuecomment-519295908